### PR TITLE
Fix PropSync and Model decorators for vue property decorator

### PR DESCRIPTION
**Pull request by yoyo930021**
For #1323 

I find some problems when doing #2141 
I write some wrong logics when use `PropSync` and `Model` decorators.

ref:
- https://github.com/kaorun343/vue-property-decorator#-propsyncpropname-string-options-propoptions--constructor--constructor---decorator
- https://github.com/kaorun343/vue-property-decorator#-modelevent-string-options-propoptions--constructor--constructor---decorator

### CODE REVIEW

File: server/src/modes/script/componentInfo.ts [hunk 1]
```diff
@@ -113,10 +113,24 @@ function getProps(tsModule: T_TypeScript, defaultExportType: ts.Type, checker: t
...
+      const prop = propSymbol.valueDeclaration as ts.PropertyDeclaration;
+      const decoratorExpr = prop.decorators?.find(decorator =>
+        tsModule.isCallExpression(decorator.expression)
+          ? propDecoratorNames.includes(decorator.expression.expression.getText())
+          : false
+      )?.expression as ts.CallExpression;
+      const decoratorName = decoratorExpr.expression.getText();
+      const args = decoratorExpr.arguments;
+
+      const firstNode = args[0];
```
Comment (octref) [inline server/src/modes/script/componentInfo.ts:126]:
```suggestion
      const firstNode = decoratorExpr.arguments[0];
```

**Review by octref — APPROVED** [review]
Good work 👍 Just a small nitpick. No need to create only-use-once variables.

**Comment by octref** [comment]
I'll just commit the small change and merge.

File: server/src/modes/script/componentInfo.ts (hunk 1 above)
Comment (yoyo930021) [reply server/src/modes/script/componentInfo.ts]: Oh, It's just use in #2166

**Pull request merged**